src/hooks/posters.ts

```diff
@@ -89,7 +91,7 @@ export const usePosters = (posterHallId: string) => {
   const searchedPosterVenues = useMemo(() => {
     const normalizedSearchQuery = searchQuery.trim();
 
-    if (!normalizedSearchQuery) return filteredPosterVenues;
```

Comment:
(Delete line below this: if (!normalosedSearchQuery) etc

Reply:
removed, moved shuffle. merged staging. 
going back from a poster to posterhall still reshuffles - and it looks like it might first show original one for a moment but then reshuffle happens - so something might indeed be "flipping" (even if just being changed from null to undefined or smth like that?)
pushed

Reply:
well, this one I think is resolved -- I will move my comment to the above one